src/tooltip.jsx

```diff
@@ -23,39 +23,61 @@ class Tooltip extends Component {
     shouldShow: false,
   }
 
-  render () {
-    const child = Children.only(this.props.children)
+  child = Children.only(this.props.children)
```

Comment:
Why change this? I worry this will only be evaluated once when this is constructed, so if the children change on a render, it won't pick up the new children.

Reply:
Fair enough, I changed it to `this.child` so that PortalPopper.onMouseOut can use it, but that can be set in `render()`. Fixed.